.pre-commit-config.yaml

```diff
@@ -21,29 +17,34 @@ repos:
       - id: check-yaml
         args: ['--allow-multiple-documents']
       - id: debug-statements
-  - repo: https://github.com/psf/black
-    rev: 21.12b0
-    hooks:
-      - id: black
-        args: ["--target-version", "py37"]
-      - id: black-jupyter
-        args: ["--target-version", "py37"]
   - repo: https://github.com/pycqa/flake8
     rev: 4.0.1
     hooks:
       - id: flake8
         additional_dependencies: ['flake8-rst-docstrings']
         args: ['--config=setup.cfg']
-  # - repo: https://github.com/pre-commit/mirrors-autopep8
-  #   rev: v1.4.4
-  #   hooks:
-  #   - id: autopep8
-  #     args: ['--global-config=setup.cfg','--in-place']
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
+    hooks:
+      - id: black
+        args: ["--target-version", "py38"]
   - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
       - id: isort
         args: ['--profile', 'black']
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.2.3
+    hooks:
+      - id: nbqa-black
+      - id: nbqa-pyupgrade
+        args: [--py38-plus]
+      - id: nbqa-isort
+#   - repo: https://github.com/kynan/nbstripout
+#     rev: 0.5.0
+#     hooks:
+#       - id: nbstripout
+#         files: ".ipynb"
```

Comment:
What happened here?

Reply:
`nbstripout` was causing issues with `black-jupyter`, due to an existing bug/feature (https://github.com/kynan/nbstripout/issues/121). I turned it off for the time being. We can probably re-enable it once the notebooks are in working order again.